.bit/responses/2.1-Week 2 Step 1.md

```diff
@@ -7,104 +7,303 @@ step: 1
 name: Week 2 Step 1
 ---
 
-## Week 2 Step 1
-### [Parsing](https://oxylabs.io/blog/what-is-data-parser#:~:text=Data%20parsing%20is%20a%20method,be%20easily%20read%20and%20understood.) Multipart
+**Week 1 Step 1** ⬤◯◯
 
-#### Setting up the Emotion Reader Azure Function
-Create **a new HTTP trigger function** in your Azure portal along with the Function App. Navigate to your Function App. **This is not the function code, but the actual app service resource**.
+🕐 **Estimated completion:** 1-2 hours
 
-> In order to parse an image, we'll be using the `parse-multipart` package. [What is the parse-multipart package?](https://www.npmjs.com/package/parse-multipart) Install it!
+## Emotion Analysis
 
-Be sure to add your **new Function url** to a secret named `FUNCTION_URL`!
+### ✅ Task:
+**Create an azure function that can send a static image to the face API that returns if a person has a beard or not.**
+- [ ] Create a new branch named `week-2` (use this branch for all tasks in week 2)
+- [ ] Create a new Azure function
+- [ ] Create a Face [API Endpoint](https://smartbear.com/learn/performance-monitoring/api-endpoints/#:~:text=For%20APIs%2C%20an%20endpoint%20can,to%20carry%20out%20their%20function.&text=The%20place%20that%20APIs%20send,lives%2C%20is%20called%20an%20endpoint.)
+- [ ] Set up the Emotion Reader Azure Function
+- [ ] Build an Azure function that makes a request to the Face API and analyzes an image
+- [ ] Take a screenshot of your Postman response for the beard 
 
-#### Time to code
-In your Azure Portal, go into your HTTP function you just created and into the index.js file. You should see starter code there that has something like `module.exports...`.
-This is the file that we will be editing.
+## ❓ If you are confused
```

Comment:
I think this is a little too overwhelming. (I'll add this in Feedback.md as well)
We should split this part into multiple sections - there's too many drop downs